### PR TITLE
navbar added

### LINKED ISSUE #4: Top Navbar
Add a navbar at the top for internal and external navigation

### PR DESCRIPTION
**Pull request by Pulkitchamp**
Added navbar. Fixes #4

### CODE REVIEW

**Comment by ion05** [comment]
Ref the issue using "Fixes #issue.no" nd post a screenshot or host ur site on GitHub pages

File: index.html [hunk 1]
```diff
@@ -1,9 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <script defer src="https://use.fontawesome.com/releases/v5.0.7/js/all.js"></script>
```
Comment (ion05) [inline index.html:4]: Put the JS code in Index.js

File: index.html [hunk 2]
```diff
@@ -13,17 +25,35 @@
 
 <body>
   <button onclick="topFunction()" id="myBtn" title="Go to top"><i class="fa fa-arrow-up"></i></button>
+
+  <div id="pulkit">
```
Comment (ion05) [inline index.html:29]: dont use ur name as a id. Use 1 that describes what the id encloses

File: index.html [hunk 3]
```diff
@@ -13,17 +25,35 @@
...
+  </div>
+</div>
+
       <div class="container main">
         <h1 class="heading">
             LIFE-SYNC
         </h1>
         <h4 class="quote">
-            LEARN~OBSERVE~CREATE
+            LEARN-OBSERVE-CREATE
```
Comment (ion05) [inline index.html:50]: Dont change this

File: index.html [hunk 4]
```diff
@@ -39,10 +69,10 @@ <h1><div class="headT"> Who Are We ?</div></h1>
                     <h2>Yuvraaj Narula</h2>
                     <h4 style="font-size: 20px;">Front-End Developer</h4>
                         <ul class="social">
-                            <li><a href="https://www.instagram.com/yuvraaj.13/" target="_blank"><i class="fa fa-instagram"></i></a></li>
-                            <li><a href="mailto:ynarula678@gmail.com""><i class="fa fa-envelope"></i></a></li>
-                            <li><a href="https://github.com/yuvraajnarula" target="_blank"><i class="fa fa-github"></i></a></li>
-                            <li><a href="https://codepen.io/codeologynerd" target="_blank"><i class="fa fa-codepen"></i></a></li>
+                            <li><a href="https://www.instagram.com/yuvraaj.13/" target="_blank"><i class="fab fa-instagram"></i></a></li>
+                            <li><a href="mailto:ynarula678@gmail.com""><i class="far fa-envelope"></i></a></li>
```
Comment (ion05) [inline index.html:73]: why did you make changes here ?

File: index.html [hunk 5]
```diff
@@ -60,10 +90,10 @@ <h4 style="font-size: 20px;">Front-End Developer</h4>
                     <h2>Aayan Agarwal</h2>
                     <h4 style="font-size: 20px;"> Back-End Developer </h4>
                     <ul class="social">
-                        <li><a href="https://www.instagram.com/aayan_14.05/" target="_blank"><i class="fa fa-instagram"></i></a></li>
-                        <li><a href="mailto:aayanagarwal05@gmail.com"><i class="fa fa-envelope"></i></a></li>
-                        <li><a href="https://www.github.com/ion05" target="_blank"><i class="fa fa-github"></i></a></li>
-                        <li><a href="https://twitter.com/14Coderz" target="_blank"><i class="fa fa-twitter"></i></a></li>
+                        <li><a href="https://www.instagram.com/aayan_14.05/" target="_blank"><i class="fab fa-instagram"></i></a></li>
```
Comment (ion05) [inline index.html:93]: Why changes here ?

File: index.html [hunk 6]
```diff
@@ -81,9 +111,9 @@ <h4 style="font-size: 20px;"> Back-End Developer </h4>
                     <h2>Gobind Anand</h2>
                     <h4 style="font-size: 20px;"> Ideator </h4>
                     <ul class="social">
-                        <li><a href="https://instagram.com/anand138.singh" target="_blank"><i class="fa fa-instagram"></i></a></li>
```
Comment (ion05) [inline index.html:84]: Same as above

File: index.html [hunk 7]
```diff
@@ -99,11 +129,14 @@ <h4 style="font-size: 20px;"> Ideator </h4>
...
+
+                    <h4 style="font-size: 20px;">Front End Developer </h4>
                     <ul class="social">
-                        <li><a href="https://www.instagram.com/pulkit_2005/" target="_blank"><i class="fa fa-instagram"></i></a></li>
-                        <li><a href="mailto:pulkit.handa2005@gmail.com"><i class="fa fa-envelope"></i></a></li>
-                        <li><a href="https://github.com/Pulkitchamp"><i class="fa fa-github"></i></a></li>
+                        <li><a href="https://www.instagram.com/pulkit_2005/" target="_blank"><i class="fab fa-instagram"></i></a></li>
+                        <li><a href="mailto:pulkit.handa2005@gmail.com"><i class="far fa-envelope"></i></a></li>
+                        <li><a href="https://github.com/Pulkitchamp"><i class="fab fa-github"></i></a></li>
+                        <li><a href="https://codepen.io/pulkitchamp" target="_blank"><i class="fab fa-codepen"></i></a></li>
```
Comment (ion05) [inline index.html:138]: Share ss here of your profile card

File: index.html [hunk 8]
```diff
@@ -113,14 +146,17 @@ <h4 style="font-size: 20px;">Developer </h4>
...
+<br>
             </div>
             <div class="footer">
         <footer>
           <button class="signup btn">Contact Us</button>
               <div class="form-modal">
                 <div class="modal-content">
                   <div class="content-wrapper">
-                    <div class="close">+</div>
+                    <div class="closehoja">+</div>
```
Comment (ion05) [inline index.html:159]: why changes here ?

File: index.html [hunk 9]
```diff
@@ -113,14 +146,17 @@ <h4 style="font-size: 20px;">Developer </h4>
             </ul>
  </body>
 <!--JS-->
+<br>
+<br>
```
Comment (ion05) [inline index.html:150]: Why have u added `<br>` after the body has been closed

**Review by ion05 — CHANGES_REQUESTED** [review]
Do the changes as requested

**Review by yuvraajnarula — CHANGES_REQUESTED** [review]
Try to write your code in a better way rather than using some gibberish terms @Pulkitchamp.

File: index.html (hunk 5 above)
Comment (Pulkitchamp) [reply index.html:93]: logo was not working earlier

File: index.html (hunk 4 above)
Comment (Pulkitchamp) [reply index.html:73]: logos were not working earlier

File: index.html (hunk 6 above)
Comment (Pulkitchamp) [reply index.html:84]: logos were not working her too

File: index.html (hunk 7 above)
Comment (Pulkitchamp) [reply index.html:138]: image
Screenshot (45)

File: index.html (hunk 4 above)
Comment (ion05) [reply index.html:73]: do the icons still work ?

File: index.html (hunk 8 above)
Comment (Pulkitchamp) [reply index.html:159]: it was interfering with .nav close

File: index.html (hunk 9 above)
Comment (Pulkitchamp) [reply index.html:150]: to leave space between the blue bar

Comment (ion05) [reply index.html:150]: You are putting it after the body has been close ... it will not imapct

File: index.html (hunk 8 above)
Comment (ion05) [reply index.html:159]: ok

**Comment by ion05** [comment]
Host your site on github pages and send me the link

**Comment by ion05** [comment]
@Pulkitchamp you could not do the changes in 2 days

File: index.html (hunk 1 above)
Comment (Pulkitchamp) [reply index.html:4]: ye aayan yuvi ne hi daala tha apni coding me i havent done anything with it

File: index.html (hunk 2 above)
Comment (Pulkitchamp) [reply index.html:29]: i will change this one

File: index.html (hunk 1 above)
Comment (Pulkitchamp) [reply index.html:4]: this js code was already there in the coding done by @yuvraajnarula

File: index.html (hunk 3 above)
Comment (Pulkitchamp) [reply index.html:50]: ok fine i will change it back

File: index.html (hunk 1 above)
Comment (ion05) [reply index.html:4]: Yes now put it in index.js

File: index.html (hunk 4 above)
Comment (Pulkitchamp) [reply index.html:73]: yes after changing it works

**Comment by ion05** [comment]
Do the rest of the changes i requested

**Review by ion05 — APPROVED** [review]

**Pull request merged**